Apps/W1/EDocumentsConnector/app/src/Common/EDocExtCustomerCard.PageExt.al

```diff
@@ -0,0 +1,23 @@
+// ------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// ------------------------------------------------------------------------------------------------
+namespace Microsoft.EServices.EDocumentConnector;
+
+using Microsoft.Sales.Customer;
+
+pageextension 6360 "E-Doc. Ext. Customer Card" extends "Customer Card"
+{
+    layout
+    {
+        addafter("Document Sending Profile")
+        {
+            field("SignUpService Participant Id"; Rec."Service Participant Id")
```

Comment:
Why is name different between vendor and customer?

Reply:
Glitch on my side...